third_party/openthread/BUILD.gn

```diff
@@ -16,10 +16,27 @@ group("openthread") {
       "${chip_root}/third_party/openthread/platforms:libopenthread-platform",
       "${chip_root}/third_party/openthread/platforms:libopenthread-platform-utils",
     ]
+  }
+}
+
+group("openthread_device") {
```

Comment:
This one is "openthread":

Reply:
I put device has a  full thread Device or minimal Thread Device and kept simply open thread for the other one has it was already name that way but sure I scan swap/rename them

Reply:
Yes, you should swap them because the names mean something in the upstream, and that meaning should be respected here. "libopenthread" is the openthread core API. "libopenthread-platform" is the platform API.